<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchBus</class>
 <widget class="QMainWindow" name="searchBus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查詢公車路線</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>輸入公車路線編號：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBusInfo">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查詢</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="busID_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>340</width>
      <height>810</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>路線</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路線名稱</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="busOutbound_table">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>640</width>
      <height>810</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>公車路線名稱：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>去程：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>回程：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="busInbound_table">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>170</y>
      <width>640</width>
      <height>810</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>站序</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>中文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文站點名稱</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>經度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>緯度</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
